core/src/main/java/io/github/tcdl/messages/Topics.java

```diff
@@ -13,32 +13,12 @@
     private final String response;
 
     @JsonCreator
-    private Topics(@JsonProperty("to") String to, @JsonProperty("response") String response) {
+    public Topics(@JsonProperty("to") String to, @JsonProperty("response") String response) {
```

Comment:
The are only two places where "response" is null)And also if you add two constructors with @JsonCreator annotation you will receive java.lang.ArrayIndexOutOfBoundsException,to be honest I don't know why?!